### PR TITLE
PODS-2672 : Navigation for new establisher from cya page is not working correctly

### PR DESCRIPTION
**Pull request by abhijeetardale**
PODS-2672 : Navigation for new establisher from cya page is not working correctly

### CODE REVIEW

File: app/utils/hstasklisthelper/HsTaskListHelperVariations.scala [hunk 1]
```diff
@@ -80,8 +80,10 @@ class HsTaskListHelperVariations(answers: UserAnswers, viewOnly: Boolean, srn: O
   protected[utils] def establishers(userAnswers: UserAnswers): Seq[SchemeDetailsTaskListSection] =
     listOfSectionNameAsLink(userAnswers.allEstablishers)
 
-  protected[utils] def trustees(userAnswers: UserAnswers): Seq[SchemeDetailsTaskListSection] =
+  protected[utils] def trustees(userAnswers: UserAnswers): Seq[SchemeDetailsTaskListSection] = {
+    println("\n\n\n\n "+userAnswers.allTrustees)
     listOfSectionNameAsLink(userAnswers.allTrustees)
```
Comment (suryabalasingh) [inline app/utils/hstasklisthelper/HsTaskListHelperVariations.scala]: remove println

File: app/navigators/TrusteesPartnershipNavigator.scala [hunk 2]
```diff
@@ -35,7 +36,7 @@ class TrusteesPartnershipNavigator @Inject()(val dataCacheConnector: UserAnswers
     if(mode == CheckMode || mode == NormalMode){
       checkYourAnswers(index, journeyMode(mode), srn)
     } else {
-      if(answers.allTrusteesAfterDelete.nonEmpty && answers.allTrustees(index).isCompleted) anyMoreChanges(srn)
+      if(answers.get(IsPartnershipCompleteId(index)).getOrElse(false)) anyMoreChanges(srn)
```
Comment (suryabalasingh) [inline app/navigators/TrusteesPartnershipNavigator.scala]: shouldn't it be isTrusteeCompleteId?

File: app/utils/hstasklisthelper/HsTaskListHelperVariations.scala (hunk 1 above)
Comment (abhijeetardale) [reply app/utils/hstasklisthelper/HsTaskListHelperVariations.scala]: removed

File: app/navigators/TrusteesPartnershipNavigator.scala (hunk 2 above)
Comment (abhijeetardale) [reply app/navigators/TrusteesPartnershipNavigator.scala]: its already used in subscription journey, creating another ticket to refactor this

**Pull request merged**